bigtable/client/table.h

```diff
@@ -56,8 +56,8 @@ class Table {
    * @param table_id the table id within the instance defined by client.  The
    *     full table name is `client->instance_name() + '/tables/' + table_id`.
    */
-  Table(std::shared_ptr<DataClient> client, std::string app_profile_id,
-        std::string const& table_id)
+  Table(std::shared_ptr<DataClient> client,
```

Comment:
Should we have overloads that take a `bigtable::TableId` for the table_id parameter?

Reply:
As discussed offline, going to stick with string for simplicity.